### DISCUSSION TITLE
Welcome to wapanel Discussions!

### DISCUSSION THREAD

**Discussion by Firstbober** [Firstbober/wapanel#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tomysurya** [comment]
hello! 
try to build this but i get an "error: transform is not amember of std"
i'm on arch linux, any help will appreciated

**Reply by fungilife** [reply]
me too same thing
